src/js/contexts/MessageContext/MessageContext.js

```diff
@@ -32,11 +32,24 @@ export const format = (options, messages) => {
       messageObj = messageObj[idPart];
     }
   });
+
   const message =
     (options.messages ? options.messages[baseId] : undefined) ||
     messageObj ||
     options.defaultMessage;
-  return message;
+
+  const { values } = options;
+
+  let newMessage = message;
+  const tokens = message?.match(/\{(.+?)\}/g);
+  tokens?.forEach((token) => {
+    const name = token;
```

Comment:
If you want to use `name` instead of `token` should probably just change line 45 to
```jsx
tokens?.forEach(name => {
```
and delete line 46

Reply:
`token` is fine when I was `console.logging` everything I changed some names to break it up but its good now!